example/src/main/scala/scalaz/example/ContravariantCoyonedaUsage.scala

```diff
@@ -538,7 +538,7 @@ object ContravariantCoyonedaUsage {
       s.foldMap{case (st, i) => recItem[BinOrd](i, sortTypeBinOrd(st))}
 
     // The drawback here is that I can’t just build a separate stack
-    // willynilly for `Binfmt'.  I have to prove at each step that *the
+    // willy nilly for `Binfmt'.  I have to prove at each step that *the
```

Comment:
Google Sheets made this (and most of the other corrections). I probably wouldn't have actively made it, but it's right.